lib/service_actor/result.rb

```diff
@@ -84,8 +84,10 @@ def method_missing(method_name, *args) # rubocop:disable Metrics/AbcSize
       value.respond_to?(:empty?) ? !value.empty? : !!value
     elsif method_name.end_with?("=")
       data[method_name.to_s.chomp("=").to_sym] = args.first
-    else
+    elsif data.key?(method_name)
       data[method_name]
+    else
+      super
```

Comment:
To ease upgrading, instead of raising right away I think it would be nice to first issue a `warn` here and have the breaking change in a major version bump.

Reply:
Agreed, please take a look at the warning message; not sure about the exact wording 🙂